components/speedreader/rust/ffi/cbindgen.toml

```diff
@@ -0,0 +1,44 @@
+# The language to output bindings in
+language = "C++"
+
+# Options for Wrapping the Contents of the Header
+header = '''
+/* Copyright (c) 2020 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */'''
+include_guard = "BRAVE_COMPONENTS_SPEEDREADER_RUST_FFI_INCLUDE_SPEEDREADER_FFI_H_"
```

Comment:
what is this used for?

Reply:
this config controls how `components/speedreader/rust/ffi/includes/speedreader_ffi.h` is auto-generated (besides the auto-generated file having to be in output dir)

`include_guard` just specifies the full guard as per chromium's naming convention, since cbindgen doesn' know about the rest of the project structure